devices/Paexo/src/Paexo.cpp

```diff
@@ -359,6 +469,70 @@ class Paexo::PaexoImpl::PaexoTorque3DSensor : public wearable::sensor::ITorque3D
     }
 };
 
+// ===========================================
+// Paexo implementation of ForceTorque6DSensor
+// ===========================================
+class Paexo::PaexoImpl::PaexoForceTorque6DSensor : public wearable::sensor::IForceTorque6DSensor
+{
+public:
+    Paexo::PaexoImpl* paexoImpl = nullptr;
+
+    PaexoForceTorque6DSensor(Paexo::PaexoImpl* impl,
+                             const wearable::sensor::SensorName name = {},
+                             const wearable::sensor::SensorStatus status = wearable::sensor::SensorStatus::Ok) // Default sensor status is Ok, to be updated using node status from iFeelDriver
+        : IForceTorque6DSensor(name, status)
+        , paexoImpl(impl)
+    {
+        // TODO: Initialization
+    }
+
+    ~PaexoForceTorque6DSensor() override = default;
+
+    void setStatus(const wearable::sensor::SensorStatus status) { m_status = status; }
+
+    bool getForceTorque6D(Vector3& force3D, Vector3& torque3D) const override
+    {
+        assert(paexoImpl != nullptr);
+
+        std::lock_guard<std::mutex> lock(paexoImpl->mutex);
+
+        // Get FT Data from front or back FT using iFeelDriver data
+        // TODO: Cleanup ftData buffer handling
+        paexoImpl->ifeelData = paexoImpl->ifeelDriver->getData();
+
+        if (this->getSensorName().find("Left") != std::string::npos)
+        {
+           iFeel::ForceTorque ftData = paexoImpl->ifeelData.FTShoeNodes[2]->getFrontFT();
+
+           force3D[0] = ftData.x;
+           force3D[1] = ftData.y;
+           force3D[2] = ftData.z;
+
+           torque3D[0] = ftData.tx;
+           torque3D[1] = ftData.ty;
+           torque3D[2] = ftData.tz;
+
+        }
+
+        if (this->getSensorName().find("Right") != std::string::npos)
+        {
+            iFeel::ForceTorque ftData = paexoImpl->ifeelData.FTShoeNodes[2]->getBackFT();
```

Comment:
if I am not wrong, we should switch left/right

Reply:
@lrapetti This is the correct order, I dropped the flipped order commit.